org.mozilla.FirefoxESR.json

```diff
@@ -0,0 +1,1955 @@
+{
+    "app-id": "org.mozilla.FirefoxESR",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "firefox",
+    "finish-args": [
+        /* X11 + XShm access */
+        "--share=ipc",
+        "--socket=x11",
+        /* Wayland access */
+        "--socket=wayland",
+        /* Needs to talk to the network: */
+        "--share=network",
+        /* Enable playing sound thru pulseaudio */
+        "--socket=pulseaudio",
+        /* Enable access to all devices (Mozilla's Firefox flatpak does this) */
+        "--device=all",
+        /* Enable themes support */
+        "--filesystem=~/.config/dconf:ro",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+        "--talk-name=org.freedesktop.FileManager1",
+        /* Picked from Mozilla configuration */
+        "--socket=pcsc",
+        "--socket=cups",
+        "--persist=.mozilla",
+        "--filesystem=xdg-download:rw",
+        "--system-talk-name=org.freedesktop.NetworkManager",
+        "--talk-name=org.a11y.Bus",
+        "--talk-name=org.gnome.SessionManager",
+        "--talk-name=org.freedesktop.ScreenSaver",
+        "--talk-name=org.gtk.vfs.*",
+        "--talk-name=org.freedesktop.Notifications",
+        "--own-name=org.mpris.MediaPlayer2.firefox.*",
+        "--own-name=org.mozilla.firefox.*",
+        "--own-name=org.mozilla.firefox_esr.*",
+        "--own-name=org.mozilla.firefox_beta.*",
```

Comment:
if firefox ESR register `org.mozilla.firefox_esr` on DBus that mean this should be the app-id. By default the `own-name` is granted for the app-id

(I assume this is actually what it register)

Reply:
Thans for your kind suggestions.

Yes, FF ESR does register `org.mozilla.firefox_esr.*`, removing that `--own-name` didn't break it (after changing the `app-id`).

I'm not certain that `org.mozilla.firefox.*`, `org.mozilla.firefox_beta.*` should be removed or not, providing [Mozilla's regular flatpak has them included](https://hg.mozilla.org/releases/mozilla-esr102/file/tip/taskcluster/docker/firefox-flatpak/runme.sh#l164).